var/www/openmediavault/js/omv/module/admin/service/docker/ContainerGrid.js

```diff
@@ -618,17 +620,16 @@ Ext.define("OMV.module.admin.service.docker.ContainerGrid", {
                 var me = this;
 
                 return [{
-                    // xtype: "textareafield",
                     xtype: "textarea",
                     name: "logs",
                     grow: false,
+                    editable: false,
                     height: 620,
-                    readOnly: true,
-                    cls: "x-form-textarea-monospaced"
-//                    fieldStyle: {
-  //                      fontFamily: "courier",
-    //                    fontSize: "12px"
-      //              }
+                    cls: "x-form-textarea-monospaced",
+                    fieldStyle: {
```

Comment:
Is there any special reason why you override the default font?

Reply:
Not really i copied the change from a very old fix when zfs plugin was in development.  Now that you mention it looks much good without monospaced.

Reply:
The x-form-textarea-monospaced is good, but i'm wondering about the fieldStyle thingy.

Reply:
I can't really remember why this was introduced I'm afraid.

Reply:
I will submit another PR with monospaced cls and no override